### DISCUSSION TITLE
project status

### DISCUSSION THREAD

**Discussion by samukweku** [h2oai/datatable#3473] [General]
hi datatable team, checking to see if the project is still being supported. It seems @oleksiyskononenko is not leading the project anymore. What's the way forward for this library? thanks @st-pasha ?

**Comment by st-pasha** [comment]
Hi Samuel, I can take a look at whatever urgent issues arise. Or what are you talking about the existing PR backlog?

**Reply by samukweku** [reply]
@st-pasha Yes the existing pr backlog. I'm just also curious about the project. Is it still ok to contribute? It just felt like everything paused and the project has been abandoned and sometimes I'm not sure if the project will still continue or be supported.

**Comment by javadba** [comment]
Over two years later and no response (and only three inconsequential commits). So that's really sad - the project is clearly definitively abandoned. It is a great project. btw is there a recommendation by the authors on what to use instead?

**Reply by samukweku** [reply]
@javadba one possible option is to fork it and build the community again?
